src/components/PostForm/components/TextArea/TextArea.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { userEvent } from '@storybook/testing-library'
+import { render, screen } from '@testing-library/react'
+import { noop } from 'lodash'
+
+import { PostFieldProps } from '../../FieldRenderer'
+
+import { TextArea } from '.'
+
+describe('TextArea', () => {
+  const fieldProps: Omit<PostFieldProps, 'onChange' | 'onBlur'> = {
+    field: {
+      key: 'textareaString',
+      title: 'Text Area',
+      type: 'string',
+      directive: 'textarea',
+      minimum: 0,
+      maximum: 1
+    },
+    value: 'default value'
+  }
+
+  it('should render all elements properly', () => {
+    render(<TextArea {...fieldProps} onChange={noop} onBlur={noop} />)
```

Comment:
Any reason why you decided to do onChange={noop} here vs onChange={onChange} like you did on line 29?

Reply:
I used jest.fn() for onChange to test that the function was properly called with a mock function, but for onBlur and onChange for the render test case, the functions aren't used so rather than mocking them it's better to show that they aren't necessary for the test cases